docs/_layouts/past-results-layout.html

```diff
@@ -92,5 +92,5 @@ <h2>{{page.subheading}} Integration Tests</h2>
   </div>
 </div>
 
-{% include_cached compatibility/past-results.html %}
+{% include compatibility/past-results.html %}
```

Comment:
Pass variables as parameters, please.

Let's continue to use the Build Cache plugin. See that plugin for instructions on how to do this.

Reply:
Done @leecalcote.